services/web/server/src/simcore_service_webserver/projects/projects_settings.py

```diff
@@ -12,3 +17,19 @@ class ProjectsSettings(BaseCustomSettings):
         default=5,
         description="defines the number of dynamic services in a project that can be started concurrently (a value of 0 will disable it)",
     )
+
+    PROJECTS_NODE_CREATE_INTERVAL: datetime.timedelta = Field(
```

Comment:
Ok I like it, but I am still not sure it deserves to be a setting. not sure how someone would change that.